pom.xml

```diff
@@ -181,6 +181,11 @@ THE SOFTWARE.
       <version>9.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci</groupId>
+      <artifactId>version-number</artifactId>
```

Comment:
Probably fine but we will see when this is integrated into core whether there is any clash with the version used by core. There are also integration tests in core which `java -jar agent.jar` so any shading/packaging issues should be smoked out.

Reply:
Core uses the same version (1.10) according to a quick GH search.